src/Stores/ArrayStore.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace MVuoncino\OpLog\Stores;
+
+use MVuoncino\OpLog\Contracts\StoreInterface;
+
+class ArrayStore implements StoreInterface
+{
+    /**
+     * @var resource
+     */
+    private $array = [];
+
+    /**
+     * @param array $record
+     */
+    public function store(array $record)
+    {
+        $this->array[] = $record;
+    }
+
+    /**
+     * @return \Generator
+     */
+    public function fetch($count = 1000)
+    {
+        for ($i = 0; $i < min($count, count($this->array)); ++$i) {
```

Comment:
I would do count($this->array) outside of the for loop, to make sure count is done only once